ipatests/test_integration/test_acme.py

```diff
@@ -542,3 +547,130 @@ def install(cls, mh):
 
         tasks.install_client(cls.master, cls.clients[0])
         tasks.install_replica(cls.master, cls.replicas[0])
+
+
+class TestACMERenew(IntegrationTest):
+
+    num_clients = 1
+
+    @classmethod
+    def prepare_acme_client(cls):
```

Comment:
This method is a copy-paste of the one already defined in TestACME. You can move it outside of the class + change the arguments to make it work:
```
def prepare_acme_client(master, client):
    # cache the acme service uri
    acme_host = f'{IPA_CA_RECORD}.{master.domain.name}'
    cls.acme_server = f'https://{acme_host}/acme/directory'

    # install acme client packages
    if not skip_certbot_tests:
        tasks.install_packages(client, ['certbot'])
    if not skip_mod_md_tests:
        tasks.install_packages(client, ['mod_md'])
```